web/modules/custom/dbcdk_community/src/Plugin/Block/ProfileBlock.php

```diff
@@ -173,6 +174,21 @@ protected function parseProfile(Profile $profile, array $fields) {
           ];
           break;
 
+        // The profile doesn't contain its roles so we cannot get the value as
+        // a regular property so we have to make a separate request for roles.
+        // The display of this field is also different than a single string
+        // value since we would like the roles to be presented as a list to make
+        // it easier overview.
```

Comment:
This was a great catch! :+1:

I have implemented the change and this should now be considered an outdated comment :)